mappings/core.ts

```diff
@@ -176,6 +176,7 @@ export function handlePunkOffered(e: PunkOffered): void {
   provenance.tenant = UserAddress.load(newTenantID).id;
   provenance.tenancyDates = tenancyDates.id;
   provenance.minSalePriceInWei = hexPackedRentData;
+  provenance.timestamp = e.block.timestamp;
```

Comment:
running `yarn graph build` tells us that we need to make a simple cast.

e.block.timestamp is '~lib/@graphprotocol/graph-ts/index/BigInt'. But provenance.timestamp is Int.

We can do two things

(1) change provenance.timestamp to BigInt or
(2) cast the BigInt into Int, which can be done by prefixing with <i32> like in line 169. Like so: <i32>e.block.timestamp.

Second option is preferable, since timestamps are not BigInts.

Reply:
Just deployed the updated subgraph. Here is the id: QmUHmTKmhGU7vZZGbQNpMaKssNURXEtpwbDyUbWoCKGgnQ. You can substitute it on line 28 in contexts/punk.js and test this branch out. Thank you

Reply:
The deployed subgraph returns empty array. I went through the code, discovered that the tenancyDates.start is the same timestamp, so modified my code to reflect that one. The sorting is not affected still, as id and block.timestamp is same ordered.

Reply:
yeah, deployed subgraph has an error somewhere: https://thegraph.com/explorer/subgraph/re-nft/cryptopunks-rent?selected=logs